simtools/constants.py

```diff
@@ -5,6 +5,5 @@
 
 # URL to the schema repository
 SCHEMA_URL = (
-    "https://gitlab.cta-observatory.org/cta-science/"
-    "simulations/simulation-model/model_parameters/-/raw/main/schema/"
+    "https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/model_parameters/"
```

Comment:
Do we still want to give it as a URL even if it is part of the project now? This way we are getting the parameter schema from the released version only. I think it would be better if we point it to the directory of the current version of simtools (whichever version one is working on). This would allow us to test implementations of the schema files on the spot. Same for all the links.

Reply:
We need to keep url, as this is what is listed in the metadata of the model parameters (model parameters are not part of simtools).

What is correct is that we do not need this URL to read the schema files, as they are part of the package. This is related to the previous point and I've opened an issue on this: #899